Dockerfile

```diff
@@ -16,6 +16,9 @@ LABEL name="openshift-ansible" \
 
 USER root
 
+# Create a symlink to /opt/app-root/src so that files under /usr/share/ansible are accessible
+RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
+
```

Comment:
I thought this would simplify how we work/debug the images as the same paths are valid.  I am going to drop this patch from the series for now.